src/wp-includes/array.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Array API: WordPress array utilities.
+ *
+ * @package WordPress
+ * @since 5.6.0
+ */
+
+/**
+ * This function allows to easily access a part from a php array.
+ * It is equivalent to want lodash get provides for JavaScript and is useful to have something similar
+ * in php so functions that do the same thing on the client and sever can have identical code.
```

Comment:
```suggestion
 * It is the PHP equivalent of JavaScript's lodash.get, and mirroring it may help other components
 * retain some symmetry between client and server implementations.
```